config/environments/development.rb

```diff
@@ -33,17 +44,23 @@
   config.eager_load = false
 
   require 'rack-webauth/test'
-  config.middleware.use(
-    Rack::Webauth::Test,
-    user: Settings.WEBAUTH.USER,
-    mail: Settings.WEBAUTH.MAIL,
-    ldapprivgroup: Settings.WEBAUTH.LDAPPRIVGROUP,
-    suaffiliation: Settings.WEBAUTH.SUAFFILIATION,
-    displayname: Settings.WEBAUTH.DISPLAYNAME,
-    ldapauthrule: Settings.WEBAUTH.LDAPAUTHRULE
-  )
+  if Settings.WEBAUTH
```

Comment:
ah, yeah, ok.  well, we need something along these lines, because we want to use Rack::Webauth::Test to fake webauth info on laptops, but not on argo-dev (where we want webauth to pass along real info).

i guess you could couple this with LN7 and LN12 to argue for a "laptop" environment?  i'm not sure whether a proliferation of environments is better than a proliferation of settings.  to me, one nice thing about settings is the that an "environment" can support slightly divergent deployments to specific instances.

i don't have particularly strong feelings on that tradeoff in general.  in this case, i'd vote for the approach taken so far on this branch (new settings instead of new environments).  laptops and argo-dev feel more similar than different (and talk to some of the same services).  and adding new settings feels lighter than adding new environments.  but i could also understand a distaste for having conditional statements in config files.